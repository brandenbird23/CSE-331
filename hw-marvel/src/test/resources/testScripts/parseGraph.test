# Test parser of graph for nfl players

LoadGraph nfl nflPlayers.csv

ListChildren nfl TOM-BRADY
ListChildren nfl PEYTON-MANNING
ListChildren nfl GENO-SMITH
ListChildren nfl GEORGE-KITTLE
ListChildren nfl DANIEL-JONES
ListChildren nfl RUSSEL-WILSON
ListChildren nfl MIKE-EVANS
ListChildren nfl ROB-GRONKOWSKI
ListChildren nfl PATRICK-MAHOMES