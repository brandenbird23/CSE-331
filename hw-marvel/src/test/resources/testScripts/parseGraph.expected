# Test parser of graph for nfl players

loaded graph nfl

the children of TOM-BRADY in nfl are: MIKE-EVANS(BUCS) PEYTON-MANNING(RETIRED) ROB-GRONKOWSKI(BUCS) ROB-GRONKOWSKI(PATRIOTS) ROB-GRONKOWSKI(RETIRED) TRISTAN-WURFS(BUCS)
the children of PEYTON-MANNING in nfl are: ROB-GRONKOWSKI(RETIRED) RUSSEL-WILSON(BRONCOS) TOM-BRADY(RETIRED)
the children of GENO-SMITH in nfl are: JORDYN-BROOKS(SEAHAWKS) RUSSEL-WILSON(SEAHAWKS)
the children of GEORGE-KITTLE in nfl are: BROCK-PURDY(49RS)
the children of DANIEL-JONES in nfl are: SAQUON-BARKLEY(GIANTS)
the children of RUSSEL-WILSON in nfl are: GENO-SMITH(SEAHAWKS) JORDYN-BROOKS(SEAHAWKS) PEYTON-MANNING(BRONCOS)
the children of MIKE-EVANS in nfl are: ROB-GRONKOWSKI(BUCS) TOM-BRADY(BUCS) TRISTAN-WURFS(BUCS)
the children of ROB-GRONKOWSKI in nfl are: MIKE-EVANS(BUCS) PEYTON-MANNING(RETIRED) TOM-BRADY(BUCS) TOM-BRADY(PATRIOTS) TOM-BRADY(RETIRED) TRISTAN-WURFS(BUCS)
the children of PATRICK-MAHOMES in nfl are:
